CMakeLists.txt

```diff
@@ -132,6 +132,11 @@ if(ENABLE_BOOST)
   endif()
 endif()
 
+# if this flag is set, Eigen will use ThreadpoolDevice with the number of thread as specified
+if (THREADPOOL) 
```

Comment:
See my previous answer. I agree with you that we can enable it as default. @neubig comments are welcome.